base/array.jl

```diff
@@ -1316,6 +1335,8 @@ Subsequent items are shifted left to fill the resulting gap.
 If specified, replacement values from an ordered
 collection will be spliced in place of the removed item.
 
+See also: [`pop!`](@ref), [`shift!`](@ref), [`insert!`](@ref)
```

Comment:
Also `deleteat!` (i.e. links should generally go both directions, right?).

Reply:
Yeah, right. I think this would be easier if `See also:` lists would be generated automatically from some kind of relation-table.